kitsune/sumo/static/sumo/js/main.js

```diff
@@ -3,7 +3,6 @@ window.k = window.k || {};
 
 (function () {
   k.LAZY_DELAY = 500;  // delay to lazy loading scripts, in ms
-  k.STATIC_URL = $('body').data('static-url');
```

Comment:
There are a couple of references in the `styleguide` too

Reply:
The styleguide has a bunch of peculiarities and I'm tackling them in a separate issue: https://github.com/mozilla/sumo-project/issues/894